packages/features/ee/organizations/pages/settings/other-team-members-view.tsx

```diff
@@ -215,7 +215,7 @@ const MembersView = () => {
                       if (Array.isArray(data.usernameOrEmail)) {
                         showToast(
                           t("email_invite_team_bulk", {
-                            userCount: data.usernameOrEmail.length,
```

Comment:
`userNameOrEmail` has all the emails that were sent. We need to count only the emails that were invited